design/specs/specs-null-values.md

```diff
@@ -74,47 +74,54 @@ As for the process of transforming values to and from NULL, it will be important
 
 1. The user edits a NULL value in a column that is set to text data type
 2. The user double-clicks the value cell and enters edit mode
-3. The user doesn't enter a value and leaves edit mode by hitting enter or clicking outside of the active cell
-4. The NULL value is replaced by an empty string
+3. The user sees a text input with a placeholder containing the word 'NULL.'
+4. The user doesn't enter a value, and the value replaces the placeholder text
```

Comment:
I don't understand what this is getting at.

Reply:
I have updated the text to clarify. It is mostly the behavior in which the cell turns blank rather than NULL if content was added and then cleared.